pkg/controller/perconaservermongodbrestore/perconaservermongodbrestore_controller.go

```diff
@@ -144,6 +144,10 @@ func (r *ReconcilePerconaServerMongoDBRestore) reconcileRestore(cr *psmdbv1.Perc
 		return status, errors.Wrapf(err, "get cluster %s/%s", cr.Namespace, cr.Spec.ClusterName)
 	}
 
+	if cluster.Spec.Unmanaged {
```

Comment:
what about backups?

Reply:
PBM doesn't work before replset is initialized. But, once the unmanaged nodes added to `externalNodes` in managed cluster I don't see any reason to prevent backups in unmanaged cluster. What do you think?

Reply:
PBM is the centralized thing for the whole MongoDB cluster